erica_app/erica/elster_xml/est_mapping.py

```diff
@@ -248,6 +250,9 @@ def check_and_generate_entries(est_data, year=2019):
     if est_data.get('person_b_idnr'):
         enriched_est_data['zusammen_veranlagung'] = 'X'
 
+    if est_data.get('telephone_number'):
+        enriched_est_data['telephone_number'] = est_data['telephone_number']
+
```

Comment:
I don't think that's necessary. we copy the whole `est_data` in l.225, so as the field does not change the name, it is already in the enriched_data as is. :)

Reply:
Thank you! :)

Reply:
Thank you! :) your right of course, I overlooked l.255 :grin: